core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQLUtils.kt

```diff
@@ -7,12 +7,34 @@ import org.evomaster.core.problem.graphql.param.GQReturnParam
 import org.evomaster.core.problem.util.ParamUtil
 import org.evomaster.core.search.gene.*
 import org.slf4j.LoggerFactory
+import java.util.*
 import javax.ws.rs.client.Entity
 
 object GraphQLUtils {
 
     private val log = LoggerFactory.getLogger(GraphQLUtils::class.java)
 
+    val history: Deque<String> = ArrayDeque<String>()
```

Comment:
a public mutable static state? this needs to be refactored

Reply:
Did not understand this one.

Reply:
Already removed.